sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentAction.cs

```diff
@@ -60,7 +60,6 @@ public AnalyzeSentimentAction(AnalyzeSentimentOptions options)
         /// targets of a product or service (also known as Aspect-Based sentiment analysis).
         /// If set to true, the returned <see cref="SentenceSentiment.Opinions"/>
         /// will contain the result of this analysis.
-        /// <para>Only available for service version v3.1-preview and up.</para>
```

Comment:
would it make sense to change this to "v3.1 and up" instead of deleting?

Reply:
This is already explained a couple of lines below in the `<remarks>` section: https://github.com/Azure/azure-sdk-for-net/pull/22417/files/23e3f86fece2bd9fcb23f153ad75da8f934ca31c#diff-8413e5add00d145447a22807b999c00d1cf1807d493cf60ccfae056437f682dfR65 I just forgot to clean it :(